loops.js

```diff
@@ -51,3 +65,12 @@
 // >> 9
 // ==========================================
 
+for (let i = 0; i < 10; i++) {
+    if (i < 3){
```

Comment:
Het is echt mierenneukerij, maar je bent niet helemaal consistent met de spaties binnen de IF condities.